src/server/app.js

```diff
@@ -27,6 +27,10 @@ if (nconf.get('NODE_ENV') !== 'production') {
   app.use(logger('tiny'));
 }
 
+app.use('/static/images/:image([A-Za-z\d\/]*)', function(req, res){
+  res.redirect('https://s3.amazonaws.com/gcc-public/static/images/' + req.params.image);
```

Comment:
@joeyzia In the future, do you think it'll be better to directly link to aws resources instead of first routing through our server?